database.cc

```diff
@@ -481,6 +486,11 @@ database::setup_metrics() {
                                        " to be able to admit new ones, if there is a shortage of permits."),
                        {streaming_label_instance}),
 
+        sm::make_derive("reads_shed_due_to_overload", _read_concurrency_sem.get_stats().total_reads_shed_due_to_overload,
```

Comment:
Should this be _streaming_concurrency_semaphore? based on the label below.

Reply:
Yes it should... Similarly, the system metrics below is also wrong. I'll fix that tomorrow

Reply:
today*